tests/test_kiwi.py

```diff
@@ -53,7 +54,8 @@ def test_kiwi_installation(auto_container):
 
 
 @pytest.mark.skipif(
-    PODMAN_SELECTED and os.geteuid(),
+    LOCALHOST.system_info.arch != "x86_64"
+    or (PODMAN_SELECTED and os.geteuid()),
     # https://github.com/containers/podman/issues/17715#issuecomment-1460227771
     reason="PODMAN requires root privileges for kiwi tests",
```

Comment:
The reason needs adjustment

Reply:
that's ambiguous - it's not a restriction of podman but of how the test is written. it is just a temporary skip, eventually I'll find the time to rework the test to be generic, but we need to get the images out.